app/src/main/java/org/shadowice/flocke/andotp/Database/Entry.java

```diff
@@ -131,6 +131,9 @@ public Entry(String contents) throws Exception {
 
         String secret = uri.getQueryParameter("secret");
         String label = uri.getPath().substring(1);
+        if(label.contains(":")) {
+            label = label.split(":")[1];
```

Comment:
I think only splitting the label if it contains a `:` is not the best way of doing this. There could potentially be cases where it contains an `:` without the issuer.
We should probably check if it starts with `<issuer>:` and remove that part specifically.

Reply:
@flocke Thanks for the suggestions. I'll make a commit towards this requested change.

Reply:
Here is how it's done in the Google Authenticator:
https://github.com/google/google-authenticator-android/blob/master/java/com/google/android/apps/authenticator/otp/AccountDb.java#L163

Reply:
@flocke @SagePtr
Please review the recent changes